velox/exec/MergeJoin.h

```diff
@@ -20,6 +20,28 @@
 #include "velox/exec/Operator.h"
 
 namespace facebook::velox::exec {
+
+/// The merge join operator assumes both streams, left (from addInput()) and
+/// right (from rightSource), are sorted in the join key.
```

Comment:
> sorted in the join key.

in -> on? 

Should we mention whether they are sorted ASC or DESC?

Reply:
that's actually a good question. Do we assume they are always sorted ASC?

Reply:
I would assume we expect inputs to be sorted ASC, but I haven't checked.